prisma/migrations/20220321202517_initial_user_and_journalentry/migration.sql

```diff
@@ -0,0 +1,33 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+    "displayName" TEXT,
+    "firstName" TEXT,
+    "lastName" TEXT,
+    "email" TEXT,
+
+    CONSTRAINT "User_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "JournalEntry" (
+    "id" SERIAL NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+    "date" TEXT NOT NULL,
+    "exercise" INTEGER NOT NULL,
+    "garlandPose" INTEGER NOT NULL,
+    "kegels" INTEGER NOT NULL,
+    "prenatalVitamins" BOOLEAN NOT NULL,
+    "probiotics" BOOLEAN NOT NULL,
+    "proteinIntake" INTEGER NOT NULL,
+    "authorId" TEXT NOT NULL,
+    "waterIntake" INTEGER NOT NULL,
+
+    CONSTRAINT "JournalEntry_pkey" PRIMARY KEY ("id")
+);
+
+-- AddForeignKey
+ALTER TABLE "JournalEntry" ADD CONSTRAINT "JournalEntry_authorId_fkey" FOREIGN KEY ("authorId") REFERENCES "User"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
Creates User and JournalEntry tables in the database
Connects the JournalEntry to the user through the authorId